README.md

```diff
@@ -80,6 +80,10 @@ A complete set of parameters can be found in `db/db_bench.cc`
 Sample usage:  
 `./db_bench --benchmarks=fillrandom,readrandom --num=1000000 --value_size=1024 --reads=500000 --db=/tmp/pebblesdbtest-1000`
 
+Use `filter` benchmark property to print the filter policy statistics like memory usage.
+
+`./db_bench --benchmarks=fillrandom,readrandom,filter --num=1000000 --value_size=1024 --reads=500000 --db=/tmp/pebblesdbtest-1000`
+
```

Comment:
Can we have an example of what the output would look like? Just the relevant lines you added, not the whole output. For example, for `./db_bench --benchmarks=fillrandom,readrandom,filter --num=1000000 --value_size=1024 --reads=500000 --db=/tmp/pebblesdbtest-1000` what would the filter output? (or something smaller would also work)

Reply:
running it against these benchmark properties gave an output of the form 

```
fillrandom   :     110.460 micros/op;    9.0 MB/s   
readrandom   :       4.120 micros/op; (5000 of 10000 found)

Filter in-memory size: 0.024 MB
Count of filters: 1928 
```

Should I just add the filter specific output lines?

Reply:
This should be good! Thanks!

Reply:
Please include the fillrandom, readrandom likes as well, it gives good context

Reply:
It has been included.

Reply:
thanks! I've approved the PR for merging.